### ISSUE TITLE
Early exit on bb shell on WSL corrupts terminal input visibility.

### ISSUE THREAD

**Issue by jsg24** [babashka/babashka#1512]
**version**
v1.2.174

**platform**
```
wsl -l -v
  NAME                   STATE           VERSION
* Ubuntu                 Running         1
```

On Windows 10.

**problem**

If I run:
```bash
bb sleep-test
```
And then once the program is running I exit early with `Ctrl C` , I cannot see what I type in my shell at all. I then have to start a new shell to see my input again.

Even if I type `clear` it doesn't solve it.

**repro**

(https://github.com/jsg24/problems/tree/main/001_bb_shell_text_gone)

**expected behavior**

Curiously, if I run this command which is exactly the same as the `sleep-test` task:
```bash
clj -X:test/cognitect
```
And exit early, no problem, I can see my input.

**Comment by seancorfield** [comment]
If you enter the command `stty sane` does it fix things? There are quite a few programs out there that can leave the terminal in that state so I've gotten fairly used to having to enter that command if I quit things...

**Comment by borkdude** [comment]
I can't repro this with WSL myself:

<img width="1017" alt="Screenshot 2023-03-08 at 19 37 36" src="https://user-images.githubusercontent.com/284934/223804831-c546e444-65b4-48e5-b2b7-0f61c189012a.png">

Sidenote: you can also run `(clojure "-X:test/cognitect")`

**Comment by jsg24** [comment]
Hi @borkdude . The clj version I have is `Clojure CLI version 1.10.3.1069` .

@seancorfield After I ran `bb sleep-test` and my input disappeared, I did run `stty sane` and it worked! I got my input visibility back!

Here's a screenshot:
![sttysaneworks](https://user-images.githubusercontent.com/1567745/223805706-7d98d826-9f30-4120-9eb2-1d9723250c1b.PNG)

I'm not sure why my version of WSL is acting this way though.
